### DISCUSSION TITLE
Running total in Dax Studio

### DISCUSSION THREAD

**Discussion by aesalasaguero** [DaxStudio/DaxStudio#1251] [Q&A]
Hi There, Could anybody held with this Dax Query. I Group Sales by month and created a raking, as following: 

DEFINE
      VAR Sales =
        ADDCOLUMNS (
            VALUES ( 'DIM PRODUCTO'[PRODUCTO]),
            "@SalesMonth", CALCULATE([VENTAS EN UNIDADES], 'DIM TIEMPO'[AÑO-MES]="2024-03")
        )
EVALUATE
ADDCOLUMNS (
    Sales,
    "Rank",
        RANKX (
            Sales,
            [@SalesMonth]
        )
	        
)
ORDER BY [@SalesMonth] DESC
This is the result:
<img width="313" alt="Result" src="https://github.com/DaxStudio/DaxStudio/assets/108351406/9882e1df-ab99-4c54-bb56-d824555f0890">

Is there any way to do a running total based on this result?

Thanks

**Comment by dgosbell** [comment]
It should not be too hard to do, But you would be better asking  this question here https://community.fabric.microsoft.com/t5/DAX-Commands-and-Tips/bd-p/DAXCommands as it's not really specific to DAX Studio and there are many other DAX experts on that forum that could help.
